Author: Stefan Fritsch <sf@debian.org>
Description: _GNU_SOURCE needs to be #defined before #including stuff
(causing FTBFS on hurd, #530286)
Index: apr-1.4.5/include/apr.h.in
===================================================================
--- apr-1.4.5.orig/include/apr.h.in	2011-05-24 16:19:06.038579344 +0400
+++ apr-1.4.5/include/apr.h.in	2011-05-24 16:19:12.317255043 +0400
@@ -33,6 +33,10 @@
  * for Win32 or Netware by those build environments, respectively.
  */
 
+#ifndef _GNU_SOURCE
+# define _GNU_SOURCE 1
+#endif
+
 /**
  * @defgroup APR Apache Portability Runtime library
  * @{
